app/controllers/order_details_controller.rb

```diff
@@ -121,7 +121,7 @@ def prevent_edit_based_on_state
   end
 
   def order_editable?
-    @order_detail.customer_editable?
```

Comment:
I think the `OrderDetail#customer_editable?` method can go away now. I haven't found it being used anywhere else.